service_discovery/dto/service_proposal.go

```diff
@@ -48,6 +50,14 @@ type ServiceProposal struct {
 	ProviderContacts []Contact `json:"provider_contacts"`
 }
 
+// SetDiscoveryData updates service proposal description with general data
+func (proposal *ServiceProposal) SetDiscoveryData(providerID identity.Identity, providerContact Contact) {
+	proposal.Format = "service-proposal/v1"
```

Comment:
can we make this a const instead of a "magic string" so we can reuse it if needed?

Reply:
defined constant